services/src/main/proto/census.proto

```diff
@@ -0,0 +1,318 @@
+// Copyright 2016, Google Inc.
```

Comment:
This proto defines the types returned by monitoring.proto. Since it's just cloned from the instrumentation-java repository, this can be replaced with a dependency (e.g., a git submodule) later.